webui/react/src/shared/utils/color.ts

```diff
@@ -24,6 +24,14 @@ interface RgbaColor {
   r: number;
 }
 
+/** CIELAB color space */
+interface CIELAB {
+  a: number;
+  b: number;
+  /** perceptual lightness */
+  l: number;
```

Comment:
nit: since it's short, maybe just put the comment on the same line as the type definition, like `ColorScale` does below

Reply:
we wouldn't get this any other way.
![Screen Shot 2022-08-03 at 15 04 35](https://user-images.githubusercontent.com/12127420/182688939-fecea7ce-6030-424c-968a-6a1da41f2975.jpg)